__tests__/main.test.ts

```diff
@@ -33,6 +70,10 @@ describe('action', () => {
 
     await main.run()
     expect(runMock).toHaveReturned()
+    expect(getOctokit).toHaveBeenCalled()
+    expect(contextInfo).toHaveBeenCalled()
+    expect(getCommit).toHaveBeenCalled()
+    expect(analyzeDiffCov).toHaveBeenCalled()
```

Comment:
This offers no value to the test, it just makes it more difficult to change the implementation.